microsim/opencl/doc/model_design.md

```diff
@@ -123,6 +127,10 @@ Buffers = namedtuple(
         "place_counts",
 
         "people_ages",
+        "people_obesity",

```

Comment:
Bit of a nit but we should update or delete the memory usage calculation in this doc too.

Reply:
yep good point, maybe we should just calculate this automatically from the buffers object in the snapshot? Would save us having to update it manually if the buffers change. However I might leave this for another PR though, what do you reckon?

Reply:
Agreed no rush, this is very low priority.